navigation/src/main/kotlin/com/amarcolini/joos/drive/AbstractDiffSwerveDrive.kt

```diff
@@ -62,10 +53,8 @@ abstract class AbstractDiffSwerveDrive(
     }
 
     override fun setDrivePower(drivePower: Pose2d) {
-        val (leftVel, rightVel) =
-            DiffSwerveKinematics.robotToWheelVelocities(drivePower, 1.0)
-        val (leftOrientation, rightOrientation) =
-            DiffSwerveKinematics.robotToModuleOrientations(drivePower, 1.0)
+        val (leftVel, rightVel) = DiffSwerveKinematics.robotToWheelVelocities(drivePower, trackWidth)
```

Comment:
Again, looking at other drive classes, 1.0 is used in place of the actual track width.